### PR TITLE
misc(downsampler): index migration job - add logging and fix scripts

### PR DESCRIPTION
**Pull request by sherali42**
**Pull Request checklist**

- [x] The commit(s) message(s) follows the contribution [guidelines](CONTRIBUTING.md) ?
- [ ] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?

- info logger is added to log number partition keys migrated
- schema scripts are fixed with new table name

### CODE REVIEW

File: spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala [hunk 1]
```diff
@@ -42,31 +52,29 @@ object DSIndexJob extends StrictLogging {
...
+    logger.info(s"Number of partitionKey written numPkeysWritten=${count} shard=${shard} hour=${prevHour}")
   }
 
   def toPartkeyRecordWithHash(pkRecord: PartKeyRecord): PartKeyRecord = {
     val hash = Option(schemas.part.binSchema.partitionHash(pkRecord.partKey, UnsafeUtils.arayOffset))
     PartKeyRecord(pkRecord.partKey, pkRecord.startTime, pkRecord.endTime, hash)
   }
 
-  private def hour(millis: Long = System.currentTimeMillis()) = millis / 1000 / 60 / 60
+  private def hour(millis: Long = System.currentTimeMillis()) = millis / 1000 / 60 / 60 - 1
```
Comment (vishramachandran) [inline spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala]: Remove `-1`

File: spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala [hunk 2]
```diff
@@ -4,18 +4,20 @@ import scala.concurrent.Await
...
+import filodb.core.{DatasetRef, Instance}
+import filodb.core.metadata.Schemas
 import filodb.core.store.PartKeyRecord
-import filodb.downsampler.BatchDownsampler._
 import filodb.downsampler.DownsamplerSettings
 import filodb.downsampler.DownsamplerSettings.rawDatasetIngestionConfig
 import filodb.memory.format.UnsafeUtils
 
-object DSIndexJob extends StrictLogging {
+object DSIndexJob extends StrictLogging with Instance {
```
Comment (vishramachandran) [inline spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala]: Dont understand why `with Instance` is needed

File: spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala (hunk 1 above)
Comment (sherali42) [reply spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala]: fixed it

File: spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala (hunk 2 above)
Comment (sherali42) [reply spark-jobs/src/main/scala/filodb.downsampler/index/DSIndexJob.scala]: for creating session provider - line no: 46. In earlier code session provider defined in BatchDownsampler object was initialized.

**Pull request merged**